ledger/acctonline_test.go

```diff
@@ -38,17 +38,10 @@ import (
 
 func commitSync(t *testing.T, oa *onlineAccounts, ml *mockLedgerForTracker, rnd basics.Round) {
 	_, maxLookback := oa.committedUpTo(rnd)
-	dcc := &deferredCommitContext{
-		deferredCommitRange: deferredCommitRange{
-			lookback: maxLookback,
-		},
-	}
-	cdr := ml.trackers.produceCommittingTask(rnd, ml.trackers.dbRound, &dcc.deferredCommitRange)
-	if cdr != nil {
+	dcc := ml.trackers.produceCommittingTask(rnd, ml.trackers.dbRound, maxLookback)
+	if dcc != nil {
 		func() {
```

Comment:
What is going on with this anonymous function being created and immediately invoked?  I don't understand what that is accomplishing here.

Reply:
Agree, this function does not make sense since these is no defer (like RAII patttern)

Reply:
fixed/removed